tests/installation/welcome.pm

```diff
@@ -78,11 +78,6 @@ sub run {
     if (sle_version_at_least('15') && check_var('DISTRI', 'sle')) {
         # On s390x there will be only one product which means there is no product selection
         return if check_var('ARCH', 's390x');
-        # Missing HPC module on ARM, should be there according to the PRD
-        if (check_var('ARCH', 'aarch64')) {
```

Comment:
We have to return in case of arm, but do not record soft-failure, so let's put it in the same way as for s390x

Reply:
true, thanks for the catch